ComposureWatch/src/index.css

```diff
@@ -9,12 +9,31 @@
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   -webkit-text-size-adjust: 100%; */
+  /* font-default: "Config";
+  font-default-weight: 500;
+  font-accent: "Config";
+  font-accent-weight: 700; */
 }
 
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
+#video-container::after {
```

Comment:
The id works, however, if you ever wanted to reuse this you could make a class. I generally prefer classes.

`.video-background::after` or something